week2/creditcardvalidator.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title></title>
+</head>
+<body>
+  <script>
+  creditcardchecker = (creditcardnum)=>{
+    let finalarray = [];
+    if(creditcardnum.length === 15){// check if odd
+      for(let i = 0; i < 15; i++){
```

Comment:
Given that the loop logic is identical. The only difference between even / odd is which index you start the calculations on. Try to refactor it so you aren't repeating yourself by duplicating the code. Try using a function.